src/Mimi.jl

```diff
@@ -14,50 +14,32 @@ using StringBuilders
 
 export
     @defcomp,
-    @modelegate,
     MarginalModel,
     Model,
-    addcomponent,
-    add_connector_comps,
-    add_dimension!,
-    addparameter,
-    compdef,
-    compdefs,
-    compinstance,
-    compkeys,
+    addcomponent,  # push!, add_component!, add_component, addcomp!, add!; could also have multiple functions like insert!, push! etc. instead of before after keywords
+    add_connector_comps, #should this be deleted?  Docs indicate we need it ...
     components,
-    connect_parameter,
+    connect_parameter, # connect!, bind, assign, link; ! yes or no; frontrunner: connect!
```

Comment:
suggestion:  `connect_param`
reason:  matches the convention for `!` because this doesn't change something that exists (unless you mean the model itself but that's not generally what we mean with `!`); matches convention of `<action>_<comp/param/var>` that we generally seem to follow

Reply:
Actually, I'd argue for `!`. I interpret that as indicating a function with side-effects, i.e., not a pure function that returns a value but leaves all else unchanged.

Reply:
Ok that might be a more consistent interpretation, maybe above should be `add_comp!` then?

Reply:
Yes.